console/src/components/UIKit/demo/Button.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+
+import { Flex } from './styles';
+import { ButtonStyles } from '../atoms/Button/Button.style';
+
+// Buttons *************************** //
+
+export const Buttons = () => (
+  <React.Fragment>
```

Comment:
No need for Fragment here.

Reply:
I forget to remove it, it's in the demo code. I'll remove it.

Reply:
I just checked my code, there are two adjacent <Flex /> elements, that's why I used Fragment.